### ISSUE TITLE
Documentation: Problems with github markdown

### ISSUE THREAD

**Issue by vincent-hugot** [ocaml-batteries-team/batteries-included#243]
Not exactly a Batteries issue, but it does affect us...

I just pushed an update to qtest's doc; I tested it using a gitwhub wiki of [my own](https://github.com/vincent-hugot/mymiscstuff/wiki), and it renders correctly. However, once pushed to [this repository](https://github.com/ocaml-batteries-team/batteries-included/blob/master/qtest/qtest.md), the section on Manipulation Pragmas is broken. This is the very same code -- apparently github uses two different renderers... yuck.

I'm sure that github will fix that eventually, but there are other problems with this format anyway (lack of table of contents for one thing), so I'll switch to something else (namely LaTeX ==> HeVeA ==> .html ; works wonders, .pdf as a bonus).

**Bottom line:** don't waste any time trying to fix qtest's doc's markdown formatting.
